.github/workflows/performance-regression.yml

```diff
@@ -0,0 +1,147 @@
+          python amazon/ionbenchmark/ion_benchmark_cli.py read -w 20 -o /home/runner/work/ion-python/ion-python/benchmarkResults/realWorldData02_read/new --io-type file --io-type buffer --format ion_text --format ion_binary --api load_dump /home/runner/work/ion-python/ion-python/testData/realWorldData02.10n
+          python amazon/ionbenchmark/ion_benchmark_cli.py read -w 20 -o /home/runner/work/ion-python/ion-python/benchmarkResults/realWorldData03_read/new --io-type file --io-type buffer --format ion_text --format ion_binary --api load_dump /home/runner/work/ion-python/ion-python/testData/realWorldData03.10n
+
+          python amazon/ionbenchmark/ion_benchmark_cli.py write -w 20 -o /home/runner/work/ion-python/ion-python/benchmarkResults/testStructs_write/new --io-type file --io-type buffer --format ion_text --format ion_binary --api load_dump /home/runner/work/ion-python/ion-python/testData/testStructs.10n
+          python amazon/ionbenchmark/ion_benchmark_cli.py write -w 20 -o /home/runner/work/ion-python/ion-python/benchmarkResults/testLists_write/new --io-type file --io-type buffer --format ion_text --format ion_binary --api load_dump /home/runner/work/ion-python/ion-python/testData/testLists.10n
+          python amazon/ionbenchmark/ion_benchmark_cli.py write -w 20 -o /home/runner/work/ion-python/ion-python/benchmarkResults/testSexps_write/new --io-type file --io-type buffer --format ion_text --format ion_binary --api load_dump /home/runner/work/ion-python/ion-python/testData/testSexps.10n
+          python amazon/ionbenchmark/ion_benchmark_cli.py write -w 20 -o /home/runner/work/ion-python/ion-python/benchmarkResults/realWorldData01_write/new --io-type file --io-type buffer --format ion_text --format ion_binary --api load_dump /home/runner/work/ion-python/ion-python/testData/realWorldData01.10n
+          python amazon/ionbenchmark/ion_benchmark_cli.py write -w 20 -o /home/runner/work/ion-python/ion-python/benchmarkResults/realWorldData02_write/new --io-type file --io-type buffer --format ion_text --format ion_binary --api load_dump /home/runner/work/ion-python/ion-python/testData/realWorldData02.10n
+          python amazon/ionbenchmark/ion_benchmark_cli.py write -w 20 -o /home/runner/work/ion-python/ion-python/benchmarkResults/realWorldData03_write/new --io-type file --io-type buffer --format ion_text --format ion_binary --api load_dump /home/runner/work/ion-python/ion-python/testData/realWorldData03.10n
+
+      # Generates performance results for the previous commit
+      - name: Checkout the master branch of the Ion Python
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }} # TODO change back to `master`
```

Comment:
Are we planning to checkout the top commit in the master branch? If so, can we set the `ref:'master' `directly?

Reply:
Sorry for missing the note. Now I get the reason we set it as ${{ github.event.pull_request.head.sha }} just for now.